Northstar.Client/mod.json

```diff
@@ -114,6 +114,17 @@
 			"RunOn": "UI"
 		},
 		{
+			"Path": "ui/ms_slider.nut",
```

Comment:
I still feel like this should be named `ns_silder.nut` even if currently it is just a bunch of helper funcs. Once we ship this file it is going to be there even if we rename it later on.

Reply:
Named "ms_slider" for "modsettings_slider".

Don't think it should be changed because it won't likely ever be used again.

Reply:
My point is if we ship `ms_slider.nut` and then rename it later on to `ns_slider.nut` `ms_slider.nut` will still be there because as far as Im aware mod managers dont delete files.

I just dont want an unused script laying around in the files.

Reply:
Depends on the mod-manager but manual install certainly won't so yes

> I just dont want an unused script laying around in the files.

100% agree

Anything that could be renamed later, should be renamed before shipping.

Reply:
The file is renamed. Consider this resolved.